src/branding/fedora.scss

```diff
@@ -0,0 +1,83 @@
+:root {
+  /*
+  Shift PF blue shades to Fedora blue shades,
+  aligning to official Fedora blues
+  */
+  --pf-v5-global--palette--blue-50: #e4f0f9;
+	--pf-v5-global--palette--blue-100: #d2e7f5;
+	--pf-v5-global--palette--blue-200: #9dcaea;
+	--pf-v5-global--palette--blue-300: #51a2da;
+	--pf-v5-global--palette--blue-400: #3c6eb4;
+	--pf-v5-global--palette--blue-500: #294172;
+	--pf-v5-global--palette--blue-600: #16223c;
+	--pf-v5-global--palette--blue-700: #090f1a;
+
+	/*
+	Primary is usually 300 and 500,
+	but bump up brightness a touch and keep buttons, links, etc. accessible
+	*/
+  --pf-v5-global--primary-color--100: color-mix(in srgb, var(--pf-v5-global--palette--blue-300), var(--pf-v5-global--palette--blue-400) 50%);
+  --pf-v5-global--primary-color--200: color-mix(in srgb, var(--pf-v5-global--palette--blue-300), var(--pf-v5-global--palette--blue-600) 50%);
+  --pf-v5-global--primary-color--300: color-mix(in srgb, var(--pf-v5-global--palette--blue-400), var(--pf-v5-global--palette--blue-500) 50%);
+
+  /* Redefine link colors using above variables */
+  --pf-v5-global--link--Color: var(--pf-v5-global--primary-color--100);
+	--pf-v5-global--link--Color--hover: var(--pf-v5-global--primary-color--200);
+	--pf-v5-global--link--Color--light: var(--pf-v5-global--palette--blue-300);
+	--pf-v5-global--link--Color--light--hover: var(--pf-v5-global--palette--blue-200);
+	--pf-v5-global--link--Color--dark: var(--pf-v5-global--palette--blue-500);
+	--pf-v5-global--link--Color--dark--hover: var(--pf-v5-global--palette--blue-700);
+	/* Special-case visited purple to jibe with Fedora's blue */
+	--pf-v5-global--link--Color--visited: #6e4a8a;
+
+	/* Redefine active colors to use the blue palette too */
+	--pf-v5-global--active-color--100: var(--pf-v5-global--palette--blue-400);
+  --pf-v5-global--active-color--200: var(--pf-v5-global--palette--blue-100);
+  --pf-v5-global--active-color--300: var(--pf-v5-global--palette--blue-300);
+  --pf-v5-global--active-color--400: var(--pf-v5-global--palette--blue-200);
+}
+
+/* Fedora doesn't ship the on-dark version of the SVG, but we can fake it */
+.logo {
+  /* Make a monochrome variant, bump up brightness and screen it against blue */
+  filter: invert(1) grayscale(1) brightness(2.5);
+  mix-blend-mode: screen;
+}
+
+/* Override the top header to have a Fedora blue */
+.pf-v5-c-page__main-section {
+  --_text: white;
+  background: var(--pf-v5-global--palette--blue-300);
+
+  .pf-v5-c-content {
+    --pf-v5-c-content--Color: var(--_text);
+  }
+
+  html.pf-v5-theme-dark & {
+    background: var(--pf-v5-global--palette--blue-400);
+  }
+}
+
+/* Have the kebab toggle inherit the text color */
+#toggle-kebab {
```

Comment:
Is this section specific to the fedora colors?  Should it not be always black?  Can you show me some example?